test/test_helper.rb

```diff
@@ -1,10 +1,17 @@
 require 'minitest/autorun'
 require 'semian'
 require 'semian/mysql2'
+require 'semian/redis'
 require 'toxiproxy'
 require 'timecop'
 require 'tempfile'
 require 'fileutils'
 
+require 'helpers/background_helper'
+
 Semian.logger = Logger.new(nil)
-Toxiproxy.populate(File.expand_path('../fixtures/toxiproxy.json', __FILE__))
+Toxiproxy.populate(File.expand_path('../helpers/toxiproxy.json', __FILE__))
```

Comment:
I think the JSON file is handy for project that have many proxies, but yes in that case a literal hash would be simpler.